formbricks/formbricks
# feat: Replace 'Show survey to % of users' slider

## What does this PR do?

Replace 'Show survey to % of users' slider

Fixes #2716

[Screencast from 03-06-24 12:20:57 AM IST.webm](https://github.com/formbricks/formbricks/assets/114493480/fa97e6e9-1b27-440b-8387-4cb51a3c4364)

## How should this be tested?

go to survey settings-> survey type should be web or app ->survey trigger->Survey Display Settings->Show survey to % of users



## Checklist



### Required


### Appreciated
